src/Core/Common/FindMaxAndMinInImage/Code.cxx

```diff
@@ -46,24 +46,19 @@ main(int argc, char * argv[])
 
     return EXIT_FAILURE;
   }
-  std::string inputFilename = argv[1];
-
-  using ReaderType = itk::ImageFileReader<ImageType>;
-  ReaderType::Pointer reader = ReaderType::New();
-
-  reader->SetFileName(inputFilename.c_str());
-  reader->Update();
+  // Read image file name
```

Comment:
As the following line is kind of self explanatory, this comment is not very useful.

Reply:
Done